cdap-ui/app/features/hydratorplusplus/services/non-store-pipeline-error-factory.js

```diff
@@ -123,30 +123,6 @@ let hasValidName = (name, cb) => {
   cb(false);
 };
 
-let hasOnlyOneSource = (myHelpers, GLOBALS, nodes, cb) => {
-  let error;
-  let countSource = 0;
-  if (!myHelpers.objectQuery(nodes, 'length')) {
-    cb(false);
-  }
-  nodes.forEach( node => {
-    if (GLOBALS.pluginConvert[node.type] === 'source') {
-      countSource++;
-    }
-  });
-  if (countSource === 0) {
```

Comment:
It will, this PR changed the validation we do to check that it should have at least 1 source